src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = sourceString
+    .split(';')
+    .map((line) => line.trim())
+    .filter((string) => string.length > 0)
+    .map((string) => string.split(':').map((part) => part.trim()))
+    .filter((element) => element[0] && element[1])
```

Comment:
The filter condition `element[0] && element[1]` ensures that both key and value are present. However, you might want to add a check to ensure that `element` has exactly two parts to avoid potential errors if the input string is malformed.